tutorials/transfer_learning/Project.toml

```diff
@@ -7,5 +7,5 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
-Flux = "0.11.5"
-julia = "1.5"
+Flux = "0.13"
```

Comment:
Can you update the README to have `☀️ v0.13`, here:

https://github.com/christiangnrd/model-zoo/blob/transfer-fix/README.md?plain=1#L97

Reply:
Done